Minikube tests fail in actions

Minikube tests fail in https://github.com/NaaVRE/NaaVRE-helm/actions/runs/21485786906/job/62007569807 despite multiple retries. Logs:

```
...
helm  --kube-context minikube --namespace naavre install  naavre naavre/ $(find values/rendered/values/templates -type f -exec echo -n " -f {}" ';') ; rm -r values/rendered/
NAME: naavre
LAST DEPLOYED: Fri Jan 30 14:49:57 2026
NAMESPACE: naavre
STATUS: deployed
REVISION: 1
NOTES:
1. Get the application URL by running these commands:
http://naavre-dev.minikube.test/argowf(/|$)(.*)/
http://naavre-dev.minikube.test/jupyter/
http://naavre-dev.minikube.test/k8s-secret-creator/1.0.0/ui/
https://naavre-dev.minikube.test/auth/
http://naavre-dev.minikube.test/naavre-catalogue-service/
http://naavre-dev.minikube.test/naavre-containerizer-service/docs/
http://naavre-dev.minikube.test/naavre-workflow-service/docs/
http://naavre-dev.minikube.test/vreapp
Helm installation succeeded
Waiting for OIDC configuration URL to be available
OIDC configuration URL is not available
Error: Process completed with exit code 1.
```

When running `bash scripts/setup-tests.sh -f values/values-deploy-minikube.yaml` locally on minikube, there are no errors.

--- a/scripts/setup-tests.sh
+++ b/scripts/setup-tests.sh
@@ -165,26 +165,14 @@ echo "Waiting for OIDC configuration URL to be available"
 timeout=700
 start_time=$(date +%s)
 while true; do
-    echo curl -k https://$MINIKUBE_HOST/auth/admin/console/
-    curl -k https://$MINIKUBE_HOST/auth/admin/console/
-    echo curl -k https://$MINIKUBE_HOST/auth/realms/$REALM/
-    curl -k https://$MINIKUBE_HOST/auth/realms/$REALM/
     if curl -k --silent --fail https://$MINIKUBE_HOST/auth/realms/$REALM/; then
         echo "OIDC configuration URL is available"
         break
     fi
     current_time=$(date +%s)
     elapsed_time=$((current_time - start_time))
-    echo "Waiting for OIDC configuration URL (${elapsed_time}s / ${timeout}s)"
-    kubectl get po -A
-    kubectl get ing -A
-    kubectl get Keycloak -A
-    kubectl get KeycloakRealmImport -A
     if [ $elapsed_time -ge $timeout ]; then
         echo "OIDC configuration URL is not available"
-        kubectl -n naavre describe keycloaks.k8s.keycloak.org naavre-keycloak
-        kubectl -n naavre describe keycloakrealmimports.k8s.keycloak.org naavre-keycloak-realm-import
-        kubectl -n ingress-nginx logs -l app.kubernetes.io/instance=ingress-nginx -l app.kubernetes.io/component=controller --tail=-1
         exit 1
     fi
     sleep 6

